CouchbaseLite.xcodeproj/project.pbxproj

```diff
@@ -6415,13 +6441,15 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 27BE3B451E4D63AF0012B74A /* CBL Swift.xcconfig */;
 			buildSettings = {
+				SYSTEM_FRAMEWORK_SEARCH_PATHS = "$(inherited) $(SDKROOT)$(SYSTEM_LIBRARY_DIR)/PrivateFrameworks";
```

Comment:
Is this cased by adding network framework?

Reply:
Yes, this is after adding Network Framework.